_data/chains/eip155-43113.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Avalanche Fuji Testnet",
+    "chain": "AVAX",
+    "network": "testnet",
+    "rpc": [
+      "https://api.avax-test.network/ext/bc/C/rpc"
+    ],
+    "faucets": [
+        "https://faucet.avax-test.network/"
```

Comment:
Minor nit—this line is indented 4 spaces and the rest of the file uses 2 spaces for indentations

Reply:
```json
    "faucets": [
        "https://faucet.avax-test.network/"
```

Should be

```json
    "faucets": [
      "https://faucet.avax-test.network/"
```